### ISSUE TITLE
Accessibility enhancement: Add missing alt text to documentation images

### ISSUE THREAD

**Issue by soundar3078** [freeCodeCamp/freeCodeCamp#60644]
I noticed some images in the documentation don’t have alt text, which affects accessibility for screen readers.

Proposed fix:
- Add descriptive alt text for each image.

**Comment by majestic-owl448** [comment]
which images is that?

**Comment by majestic-owl448** [comment]
I am closing this as not reproducible, if you can add more details we can reopen the issue and triage it again
